tests/core/pyspec/eth2spec/test/altair/unittests/networking/test_networking.py

```diff
@@ -0,0 +1,47 @@
+from eth2spec.test.context import (
+    with_altair_and_later,
+    spec_state_test,
+)
+from eth2spec.test.helpers.state import (
+    transition_to,
+)
+
+
+@with_altair_and_later
+@spec_state_test
+def test_get_sync_subcommittee_pubkeys_current_sync_committee(state, spec):
+    # Transition to the head of the next period
+    transition_to(spec, state, spec.SLOTS_PER_EPOCH * spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD)
```

Comment:
this assumes we are aligned to `EPOCHS_PER_SYNC_COMMITTEE_PERIOD`, which we are, but i don't know if that is an explicit guarantee of our testing infra... i've seen it used enough places that perhaps it is something we (I) can rely on :)